easybuild/easyblocks/t/tbb.py

```diff
@@ -224,7 +241,10 @@ def sanity_check_step(self):
                 os.path.join('lib', 'cmake', 'TBB', 'TBBConfigVersion.cmake'),
             ])
 
-        super(EB_tbb, self).sanity_check_step(custom_paths=custom_paths)
+        if self.cfg['with_python']:
+            custom_commands.extend(['python -c "import tbb"'])
```

Comment:
In theory, the `import tbb` could succeed even without having that directory (via `$HOME/.local`, for example), so also add the `python` subdirectory to `custom_paths`:

```python
custom_paths['dirs'].append(self.tbb_subdir, 'python')
```


The code defining `self.tbb_subdir` should be moved from `make_module_extra` to the easyblock constructor, so it can be used both in `sanity_check_step` and `make_module_extra`, without causing trouble when `--module-only` or `-sanity-check-only` is used (code in `__init__` always runs, but specific `_step` methods are skipped under certain conditions)